operations/_scripts/generate/generate_provider.sh

```diff
@@ -17,8 +17,17 @@ terraform {
       source  = \"hashicorp/aws\"
       version = \"~> 4.30\"
     }
-
+    random = {
+      source  = \"hashicorp/random\"
+      version = \">= 2.2\"
+    }
+    # do we need db to be created and priviliges to be granted ? 
```

Comment:
What's the commented out bit here?

Reply:
Something @mickmcgrath13 left?
I think we can get rid of it completely